frontend/src/pages/RegistrationsPage.tsx

```diff
@@ -0,0 +1,112 @@
+import {Box, Button, TextField} from "@mui/material";
+import "./RegistrationsPage.css";
+import {FormEvent, useState} from "react";
+
+import axios from "axios";
+import {toast, ToastContainer} from "react-toastify";
+
+export default function RegistrationsPage() {
+
+    const [userEmail, setUserEmail] = useState<string>("");
+    const [userName, setUserName] = useState<string>("");
+    const [userPassword, setUserPassword] = useState<string>("");
+    const [userRepeatPassword, setUserRepeatPassword] = useState<string>("");
+    const [, setEmailError] = useState<string>("");
+    const [, setPasswordError] = useState<string>("");
+    const [, setLoading] = useState(true || false);
+
+    const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+
+        event.preventDefault();
+
+
+        if (emailValidation(userEmail) && validatePasswords(userPassword, userRepeatPassword)) {
+            setLoading(true)
+
+            const newUser = {
+                "email": userEmail,
+                "name": userName,
+                "password": userPassword
+            }
+
+            axios.post("/auth/register", newUser)
+                .then((reponse) => reponse.data)
+                .then((data) => data)
+                .catch(error => {
+                    if (error.response?.message !== undefined) {
+                        toast(error.response.message);
+                        console.log(error)
+                    }
+                })
+                .finally(() => setLoading(false));
+
+        }
+
+
+    }
+
+    const emailValidation = (userMail: string) => {
```

Comment:
vielleicht macht es Sinn Paswort in Backend zu validieren? Mit Spring Boot Validation? Nur als Idee:-)